src/Components/Common/DropDown/style.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import {
+  StyledDropDownButtonProp,
+  StyledDropDownItemProp,
+  StyledLabelProp,
+} from './type';
+
+export const StyledDropDown = styled.div`
+  position: relative;
+  display: inline-block;
+`;
+
+export const StyledLabel = styled.span<StyledLabelProp>`
+  position: absolute;
+  left: 8px;
+  top: -10px;
+  color: ${({ $labelTextColor }) => $labelTextColor};
+  font-size: ${({ $labelTextSize }) => $labelTextSize};
+  background-color: ${({ $backgroundColor }) => $backgroundColor};
+  padding: 0px 5px;
+  font-weight: 900;
+`;
+
+export const StyledDropDownButton = styled.button<StyledDropDownButtonProp>`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: ${({ $width }) => $width};
+  height: ${({ $height }) => $height};
+  background-color: ${({ $backgroundColor }) => $backgroundColor};
+  color: ${({ $textColor }) => $textColor};
+  padding: 10px 12px;
+  font-size: ${({ $textSize }) => $textSize};
+  border: 2px solid black;
+  border-radius: 1rem;
+  gap: 12px;
```

Comment:
```suggestion
  gap: 1.2rem;
```
P5: gap 뿐만 아니라 padding도 px보다는 rem으로 변환하여 맞출 수 있다면 다 맞추는 것이 좋아보입니다!